server/repository/src/db_diesel/name_row.rs

```diff
@@ -212,8 +217,11 @@ impl<'a> NameRowRepository<'a> {
         Ok(())
     }
 
+    /// Soft delete a name record (set is_active to false)
     pub fn delete(&self, name_id: &str) -> Result<(), RepositoryError> {
```

Comment:
Wonder if we should rename these soft delete method to `soft_delete()`

Reply:
Yeah, would be nice to rename to match what the function actually does